src/richie/apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -10,11 +10,11 @@
     {% endif %}
 {% endblock meta_index_rules %}
 
-
-{% block meta_opengraph_contextuals %}
-    <meta property="og:title" content="{{ current_page.get_title|truncatechars:65 }}">
-    <meta property="og:url" content="{{ SITE.web_url }}{{ current_page.get_absolute_url }}">
-{% endblock meta_opengraph_contextuals %}
+{% block meta_rdfa_context %}
+    <meta property="og:type" content="website" />
```

Comment:
I would say no because it is more like a syllabus and the interesting content is the course itself. And we wouldn't expect to find it in our news reader?

Reply:
All right!